src/pretix/control/utils.py

```diff
@@ -0,0 +1,201 @@
+                logger.error(f"Rate limit error during {operation_name}: %s", str(e))
+                raise ValidationError(
+                    f"Too many requests. Please try again later: {getattr(e, 'user_message', str(e))}"
+                )
+            except stripe.error.InvalidRequestError as e:
+                logger.error(
+                    f"Invalid request error during {operation_name}: %s | Param: %s",
+                    str(e),
+                    getattr(e, "param", "N/A"),
+                )
+                raise ValidationError(f"Invalid request: {e.user_message}")
+            except stripe.error.SignatureVerificationError as e:
+                logger.error(
+                    f"Signature verification failed during {operation_name}: %s", str(e)
+                )
+                raise ValidationError(
+                    f"Webhook signature verification failed: {str(e)}"
+                )
+            except stripe.error.PermissionError as e:
+                logger.error(f"Permission error during {operation_name}: %s", str(e))
+                raise ValidationError(
+                    f"Permission denied: {getattr(e, 'user_message', str(e))}"
+                )
+            except stripe.error.IdempotencyError as e:
+                logger.error(f"Idempotency error during {operation_name}: %s", str(e))
+                raise ValidationError(
+                    f"Idempotency error: {getattr(e, 'user_message', str(e))}"
+                )
+            except stripe.error.StripeError as e:
+                logger.error(f"Stripe error during {operation_name}: %s", str(e))
+                raise ValidationError(
+                    f"Payment processing error: {getattr(e, 'user_message', str(e))}"
+                )
+            except Exception as e:
+                logger.error(
+                    f"Unexpected error during {operation_name}: %s",
+                    str(e),
+                    exc_info=True,
+                )
+                raise ValidationError(f"An unexpected error occurred: {str(e)}")
+
+        return wrapper
+
+    return decorator
+
+
+@handle_stripe_errors("create_setup_intent")
+def create_setup_intent(customer_id: str) -> str:
+    stripe_setup_intent = stripe.SetupIntent.create(
+        customer=customer_id,
+        payment_method_types=["card"],
+        usage="off_session",
+    )
+    OrganizerBillingModel.objects.filter(stripe_customer_id=customer_id).update(
+        stripe_setup_intent_id=stripe_setup_intent.id
+    )
+    return stripe_setup_intent.client_secret
+
+
+def get_stripe_customer_id(organizer_slug: str) -> str:
+    organizer = Organizer.objects.get(slug=organizer_slug)
+    billing_settings = OrganizerBillingModel.objects.filter(
+        organizer_id=organizer.id
+    ).first()
+
+    if billing_settings and billing_settings.stripe_customer_id:
+        return billing_settings.stripe_customer_id
+    else:
```

Comment:
You don't need to write `else` keyword here, because the `if` branch makes function return.